.teamcity/src/main/kotlin/util/AdHocPerformanceScenario.kt

```diff
@@ -48,6 +49,14 @@ abstract class AdHocPerformanceScenario(os: Os) : BuildType({
             allowEmpty = false,
             description = "Which performance test to run. Should be the fully qualified class name dot (unrolled) method name. E.g. org.gradle.performance.regression.java.JavaUpToDatePerformanceTest.up-to-date assemble (parallel true)"
         )
+        text("testJavaVersion", "8", display = ParameterDisplay.PROMPT, allowEmpty = false, description = "The java version to run the performance tests, e.g. 8/11/17")
+        select(
```

Comment:
If we use this select box in a third place I think it would be time to extract it to an extension.